### PR TITLE
Added LongTextRenderer and Scrollbar to render lengthy texts

### PR DESCRIPTION
**Pull request by Sujana2422**

### CODE REVIEW

**Comment by jfboeve** [comment]
Hi @sbokku-mlogical,

Thanks for submitting a new component. There are however some points I need you to adjust / fix:

1. Please change this PR to dev according to [Contributing guidelines](https://github.com/rdkcentral/Lightning-ui#contributing)
2. Since we don't have a ScrollBar component yet, It would be nice if it could also work in combination with other components for example like a Grid. The easiest way to do this is only working with an amount of steps and maximum amount of steps. So the LongTextRenderer would have to calculate how many steps are available. Please look at the [ProgressBar](https://github.com/rdkcentral/Lightning-ui/blob/main/src/ProgressBar.js) component to see how customisation is handled there!.
3. Regarding the LongTextRenderer there will probably a bunch of changes so I won't go too far into that, please change the setDescription setter name to just description. Also there might need to change something if the text is not long enough in order to scroll.
4. Both components don't really need a _state function.

File: docs/LongTextRenderer.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+###setDescription
+
+To render the lengthy paragraphs, you can set the paragraph text using `setDescription` and focus the longTextRenderer component after 1sec as shown in below example.
+```js
+    _active() {
+        this._setState('Loading')
+        let termsAndConditions = Language.translate('termsAndConditionsDescription')
+        termsAndConditions = termsAndConditions.replace(/\n/g, '\n\n')
+        this.tag('Description').setDescription = termsAndConditions
+        setTimeout(() => {
```
Comment (gvamshi-metrological) [inline docs/LongTextRenderer.md:91]: can we handle this directly in the `LongTextRenderer` component?

File: docs/LongTextRenderer.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+          this._setState('Active')  // focus longTextRenderer
+        }, 1000)
```
Comment (gvamshi-metrological) [inline docs/LongTextRenderer.md]: can we handle focus after 1 sec, directly in the `LongTextRenderer` component?

**Comment by Sujana2422** [comment]
Hi @jfboeve, @gvamshi-metrological,

Thanks for your comments!

I have addressed the review comments and committed the code changes. Could you please verify the changes and let me know if any further changes are required?

**Comment by jfboeve** [comment]
Hi,

Sorry for the long delay, upon further inspection of the code we have decided that we are not going to accept this PR.

There are syntax concerns and api naming that do not fall inline with the rest of the library. We also have performance concerns regarding the long text renderer that will cause issues on low performance platforms.

We acknowledge the need for these components and we already have something in the works for the next quarterly release.

**Pull request closed without merging**